src/main/java/com/example/bookshop/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.example.bookshop.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
+)
+public class MapperConfig {

```

Comment:
You can use Mapper config directly in BookMapper with Mapper annotation 
(optional)

Reply:
not fixed